### PR TITLE
Wendy/eng 2472 create component helper func

### PR DESCRIPTION
**Pull request by wendybujalski**
Added a Cypress helper function for creating components

### CODE REVIEW

File: app/web/cypress/e2e/modelling-functionality/create-component.cy.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ Cypress._.times(SI_CYPRESS_MULTIPLIER, () => {
     beforeEach(function () {
       //cy.setupVariables();
       cy.loginToAuth0(AUTH0_USERNAME, AUTH0_PASSWORD);
+      cy.log("fuck yeah i logged in!");
```
Comment (johnrwatson) [inline app/web/cypress/e2e/modelling-functionality/create-component.cy.ts:16]: Probably not required!

File: app/web/cypress/e2e/modelling-functionality/create-component.cy.ts [hunk 2]
```diff
@@ -30,19 +31,22 @@ Cypress._.times(SI_CYPRESS_MULTIPLIER, () => {
...
 
-      cy.wait(5000);
+      // // drag to the canvas
+      // cy.dragTo('@region', '@konvaStage');
 
-      //check to make sure a component has been added to the outliner
-      cy.get('[class="component-outline-node"]', { timeout: 30000 }).contains('Region', { timeout: 30000 }).should('be.visible');
+      // cy.wait(5000);
+
+      // //check to make sure a component has been added to the outliner
```
Comment (johnrwatson) [inline app/web/cypress/e2e/modelling-functionality/create-component.cy.ts:48]: nit: I think it's totally cool to remove the original lines, they aren't really providing value anymore after you got it working! :)

**Review by johnrwatson — APPROVED** [review]
Looks good, minor nits really

File: app/web/cypress/e2e/modelling-functionality/create-component.cy.ts (hunk 1 above)
Comment (wendybujalski) [reply app/web/cypress/e2e/modelling-functionality/create-component.cy.ts:16]: ah whoops i definitely meant to remove that 😅

**Comment by wendybujalski** [comment]
bors merge

**Pull request merged**